src/signals/incident/components/form/ContainerSelect/components/Selector/index.tsx

```diff
@@ -215,7 +214,7 @@ const Selector = () => {
                 onClose={handleLegendCloseButton}
                 variant={panelVariant}
                 title="Legenda"
-                items={meta.featureTypes.map(featureType => ({
+                items={meta.featureTypes.filter(({ label }) => label !== 'Onbekend').map(featureType => ({
```

Comment:
You can use the `unknownFeatureType` defined in the beginning of the file to get this label

Reply:
The `unknownFeatureType` is moved to the metadata so I removed from here!